text/0001-transform-attribute-meta-parameter.md

```diff
@@ -1,8 +1,11 @@
 ---
-Start Date: 2014-08-14
-RFC PR: https://github.com/emberjs/rfcs/pull/1
```

Comment:
Losing some historical data on the left here, where other fields were filled out.

Reply:
so this idea of an "Ember Issue" doesn't look like something that we do much nowadays right? How would you feel about keeping the information about the issues but just in the body of the RFC rather than exposing it in the frontmatter? I'm just concerned about supporting something that we aren't using any more

Reply:
This might be a bad idea, but to keep this moving forward, could we stick this in an "additional-data" section?

Reply:
I think that would be great -- I just don't want to lose what we put there.

Reply:
Basically, we just need to mark them as non-standard fields. The UI could choose to show them in some generic way if we wanted, but I agree about not wanting to lose information.

Reply:
this is now fixed. I've methodically gone through every single field (with my little script) and made sure that it is represented in a `meta` object (essentially just stuff that we will ignore but we're keeping for reference) 👍